pkg/clients/networking.go

```diff
@@ -36,6 +36,11 @@ import (
 	"sigs.k8s.io/cluster-api-provider-openstack/pkg/metrics"
 )
 
+// PortExt is the base gophercloud Port with extensions used by PortStatus.
+type PortExt struct {
+	ports.Port
+}
```

Comment:
Looks redundant at the moment, but I get the point.

Reply:
this was a leftover, i removed it.